src/main/java/org/kuali/kfs/fp/document/dataaccess/impl/GeneralLedgerTransferEntryDaoOjb.java

```diff
@@ -0,0 +1,162 @@
+            criteria.addEqualTo(KFSPropertyConstants.TRANSACTION_ENTRY_SEQUENCE_NUMBER,
+                gltSourceAccountingLine.getTransferTransactionLedgerEntrySequenceNumber().toString());
+        } else {
+            // Not including UNIVERSITY_FISCAL_YEAR and UNIVERSITY_FISCAL_PERIOD_CODE because this could process old
+            // documents
+            criteria.addEqualTo(KFSPropertyConstants.PROJECT_CODE,
+                    StringUtils.isEmpty(gltSourceAccountingLine.getProjectCode())
+                ? KFSConstants.getDashProjectCode() : gltSourceAccountingLine.getProjectCode());
+
+            if (StringUtils.isEmpty(gltSourceAccountingLine.getOrganizationReferenceId())) {
+                // CUMod KFSPTS-19396: change to addIsNull criteria instead of addEqualTo empty String as that does not work in Oracle
+                criteria.addIsNull(KFSPropertyConstants.ORGANIZATION_REFERENCE_ID);
+            } else {
+                criteria.addEqualTo(KFSPropertyConstants.ORGANIZATION_REFERENCE_ID,
+                    gltSourceAccountingLine.getOrganizationReferenceId());
+            }
+
+            criteria.addEqualTo(KFSPropertyConstants.TRANSACTION_LEDGER_ENTRY_AMOUNT,
+                gltSourceAccountingLine.getAmount().toString());
+            criteria.addEqualTo(KFSPropertyConstants.TRANSACTION_DEBIT_CREDIT_CODE,
+                    gltSourceAccountingLine.getDebitCreditCode());
+            criteria.addEqualTo(KFSPropertyConstants.REFERENCE_FIN_DOCUMENT_TYPE_CODE,
+                gltSourceAccountingLine.getReferenceTypeCode());
+            criteria.addEqualTo(KFSPropertyConstants.REFERENCE_FINANCIAL_SYSTEM_ORIGINATION_CODE,
+                    gltSourceAccountingLine.getReferenceOriginCode());
+            criteria.addEqualTo(KFSPropertyConstants.REFERENCE_FINANCIAL_DOCUMENT_NUMBER,
+                    gltSourceAccountingLine.getReferenceNumber());
+            criteria.addEqualTo(KFSPropertyConstants.TRANSACTION_LEDGER_ENTRY_DESC,
+                gltSourceAccountingLine.getFinancialDocumentLineDescription());
+            criteria.addEqualTo(KFSPropertyConstants.FINANCIAL_DOCUMENT_TYPE_CODE,
+                KFSConstants.FinancialDocumentTypeCodes.GENERAL_LEDGER_TRANSFER);
+            criteria.addEqualTo(KFSPropertyConstants.FINANCIAL_SYSTEM_ORIGINATION_CODE, KFSConstants.ORIGIN_CODE_KUALI);
+            criteria.addEqualTo(KFSPropertyConstants.DOCUMENT_NUMBER, gltSourceAccountingLine.getDocumentNumber());
+        }
+
+        return criteria;
+    }
+
+    public boolean hasTransferredEntries(String documentNumber) {
+        Criteria criteria = new Criteria();
+        criteria.addEqualTo(KFSPropertyConstants.DOCUMENT_NUMBER, documentNumber);
+        criteria.addEqualTo(KFSPropertyConstants.FINANCIAL_SYSTEM_ORIGINATION_CODE, KFSConstants.ORIGIN_CODE_KUALI);
+        criteria.addNotNull(KFSPropertyConstants.GENERAL_LEDGER_TRANSFER_DOCUMENT_NUMBER);
```

Comment:
is this also a CU customization?  If so, should we add a note here as well?

Reply:
This is not CU customization, the only change to this file is the line I have marked.